core/src/main/javascript/crypto/EccCryptoContext.js

```diff
@@ -21,10 +21,22 @@
  * @implements {ICryptoContext}
```

Comment:
Update the documentation to match the new functionality (and the old one which wasn't fully documented). See EccCryptoContext.java for an example.